Source/Revit.IFC.Common/Utility/DirectoryUtil.cs

```diff
@@ -41,7 +41,7 @@ public static string IFCSchemaLocation
       {
          get
          {
-            return Path.Combine(RevitProgramPath, "EDM");
+            return Path.Combine(RevitProgramPath, "IFC");

```

Comment:
This one cannot be changed yet. The IFC schema files are located in EDM folder. The IFC folder is used only with the ODA toolkit

Reply:
@WawanSolihin Yes, I noticed this. when I worked with a 2022 branch I replaced this code with following:



<!--StartFragment-->
#if IFC_OPENSOURCE
                return Path.Combine(RevitProgramPath, "IFC");
            #endif            
                return Path.Combine(RevitProgramPath, "EDM");
            <!--EndFragment-->

Reply:
This does not really help because the next update if we integrated the ODA toolkit to 2022.1 and we need to update the open source, the condition is not useful. Also it is the Open source that uses "EDM" folder currently. The "IFC" folder is for ODA toolkit. The lines above seems to be the other way round